Header header
uint16[3] wrtAnchors 
float32[2] curr_pos
